ArticleTemplates/assets/js/modules/outbrain.js

```diff
@@ -11,7 +11,7 @@ define(function () {
     function isNonCompliant() {
         var prose = document.querySelector('.article__body > div.prose ');
 
-        if (prose && prose.lastElementChild && prose.lastElementChild.classList.contains('creative-container')) {
+        if (prose && prose.lastElementChild && GU.opts.hasEpic) {
```

Comment:
Do you still need to check `prose && prose.lastElementChild`? (or indeed, do you still need `prose` at all)